benchmarks/benchmarks/bench_reduce.py

```diff
@@ -82,6 +83,17 @@ def setup(self, dtype):
     def time_argmax(self, dtype):
         np.argmax(self.d)
```

Comment:
I wonder why the enormous speedup for this benchmark. It only has one element.

Reply:
no, it has 200k elements
https://github.com/numpy/numpy/blob/3c39c06d69dc4c2938efba481914153dd3d2e110/benchmarks/benchmarks/bench_reduce.py#L80-L81

Reply:
Ahh, right. Sorry for the noise